explainer.md

```diff
@@ -300,19 +390,23 @@ prompts a suggestion on VoiceOver for iOS
 to perform swipe gestures to increment or decrement, 
 but there is no way to handle that semantic event via any web API.
 
-The new UI Events would give web developers a mechanism 
-to listen for accessible actions directly, 
-by adding event listeners on an `Element`.
+Developers will be able to listen for keyboard events
+or input events to capture that semantic event.
 
-For example, to implement a custom slider, the author could simply listen for
-`accessibleincrement` and `accessibledecrement` events.
+For example, to implement a custom slider,
+the author could handle the `Up` and `Down` key events
+as recommended in the [ARIA Authoring Practices guide](https://www.w3.org/TR/wai-aria-practices-1.1/#slider_kbd_interaction),
+and this would handle the assistive technology event as well.
 
 ```js
-customSlider.addEventListener('accessibleincrement', function() {
-  customSlider.value += 1;
-});
-customSlider.addEventListener('accessibledecrement', function() {
-  customSlider.value -= 1;
+customSlider.addEventListener('keydown', (event) => {
+  switch (event.code) {
+  case "ArrowUp":
```

Comment:
Because iOS doesn't send keyboard events to non-native controls, would the synthetic up/down arrow key events still work in this case? Or would it be better to use the InputEvents for increment/decrement?

Reply:
Good question.